src/selfplay/loop.cc

```diff
@@ -132,6 +135,52 @@ void SelfPlayLoop::SendTournament(const TournamentInfo& info) {
   res += " draw " + std::to_string(info.results[1][0]) + " " +
          std::to_string(info.results[1][1]);
   SendResponse(res);
+
+  // Compute player1's elo relative to player 2.
+  auto wins = info.results[0][0]+info.results[0][1];
+  auto losses = info.results[2][0] + info.results[2][1];
+  auto draws = info.results[1][0] + info.results[1][1];
+  auto n = wins + losses + draws;
+  if (n == 0) {
+    // no games
+    return;
+  }
+  double w, l, d;
+  w = wins / (double)n;
+  l = losses / (double)n;
+  d = draws / (double)n;
+  // mu is the estimate of the expected score
+  auto mu = w + d / 2.0;
+  auto devW = w * pow(1.0 - mu, 2);
```

Comment:
What are `dev*` variables? Could you add a comment.
Also, according to style guide, they should be `dev_w` etc not `devW`.